canonical-web-and-design/jaas-dashboard
# Update Models listing styling

## Done

Update Models listing styling

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check user icon dropdown and logout button
- Sanity check code

Fixes #937
